documentation/book/con-custom-resources-status.adoc

```diff
@@ -48,7 +86,8 @@ status:
 ----
 <1> Status `conditions` describe criteria related to the status that cannot be deduced from the existing resource information, or are specific to the instance of a resource.
 <2> The `Ready` condition indicates whether the Cluster Operator currently considers the Kafka cluster able to handle traffic.
-<3> The `listeners` describe the current Kafka bootstrap addresses by type.
+<3> The `observedGeneration` indicates the generation of the `Kafka` custom resource that was last reconciled by the Cluster Operator.
```

Comment:
@laidan6000 is Cluster Operator a proper noun? Wondering about capitalizing operator.

Reply:
Yes, the current Streams docs use "Cluster Operator" as a proper noun throughout.